static/js/app.js

```diff
@@ -4,16 +4,17 @@ $(function () {
             "": "navigateEmpty",
             "/?marker=:id&start_date=:start&end_date=:end&show_fatal=:showFatal&show_severe=:showSevere&show_light=:showLight&show_inaccurate=:showInaccurate&zoom=:zoom&lat=:lat&lon=:lon": "navigate"
         },
-        navigate: function (id, start, end, showFatal, showSevere, showLight, showInaccurate, zoom, lat, lon) {
-            app.model.set("currentMarker", parseInt(id));
-            app.model.set("dateRange", [new Date(start), new Date(end)]);
-            app.model.set("showFatal", showFatal);
-            app.model.set("showSevere", showSevere);
-            app.model.set("showLight", showLight);
-            app.model.set("showInaccurateMarkers", showInaccurate);
-            app.map.setZoom(parseInt(zoom));
-            app.map.setCenter(new google.maps.LatLng(lat, lon));
-        },
+        //removed navigate function because we don't use backbone navigation for now
```

Comment:
@danielhers I fixed everything except of the history. I looked at Google Maps and they save the history event when mooving to a new location and not only when seaching.
I am not sure how to implement it, because from what I remember, we don't use Backbone router.